extensions/control-plane/api/management-api/contract-definition-api/src/main/java/org/eclipse/edc/connector/api/management/contractdefinition/ContractDefinitionApiExtension.java

```diff
@@ -61,7 +69,9 @@ public String name() {
     @Override
     public void initialize(ServiceExtensionContext context) {
         var jsonFactory = Json.createBuilderFactory(Map.of());
-        transformerRegistry.register(new JsonObjectFromContractDefinitionTransformer(jsonFactory));
+        var mapper = typeManager.getMapper(JSON_LD);
+        mapper.registerSubtypes(AtomicConstraint.class, LiteralExpression.class);
```

Comment:
Is this registration needed? in this context (`ContractDefinition`) there shouldn't be any relationship with the `policy` model

Reply:
Addressed